docs/content/advanced/backend/migrations.md

```diff
@@ -0,0 +1,85 @@
+# Migrations
+
+In this document, you’ll learn about what Migrations are, their purpose, how you can run them, and how you can create your own Migrations.
+
+:::note
+
+Medusa’s Migrations do not work with SQLite databases. They are intended to be used with PostgreSQL databases, which is the recommended Database for your Medusa production server.
+
+:::
+
+## Overview
+
+Migrations in Medusa are scripts that are used to make additions or changes to your database schema. They are essential for both when you first install your server and for subsequent server upgrades later on.
```

Comment:
suggestion: Would make this more general, since migrations in Medusa are no different from migrations in other contexts. I would suggest something like Typeorm's definition: `A migration is just a single file with sql queries to update a database schema and apply new changes to an existing database.`